app/components/govuk_component/header_component.rb

```diff
@@ -52,6 +55,15 @@ def container_classes
     combine_classes(%w(govuk-header__container govuk-width-container), custom_container_classes)
   end
 
+  def crown_fallback_image_attributes
+    {
+      class: "govuk-header__logotype-crown-fallback-image",
+      width: "36",
+      height: "32",
+      "xlink:href" => "",
```

Comment:
Looks like we can drop this now that we're using `<img>` with IE8 conditional comments.

```suggestion
```